src/applications/hca/tests/unit/utils/validation.unit.spec.js

```diff
@@ -30,330 +31,249 @@ describe('hca `validateServiceDates` form validation', () => {
     },
   });
 
-  context('when form data is valid', () => {
-    const spy = sinon.spy();
-    const { errors, fieldData, formData } = getData({ spy });
+  afterEach(() => {
+    dischargeDateSpy.reset();
+    entryDateSpy.reset();
+  });
 
-    it('should not set error message', () => {
-      validateServiceDates(errors, fieldData, formData);
-      expect(spy.called).to.be.false;
-    });
+  it('should not set error message when form data is valid', () => {
+    const { errors, fieldData, formData } = getData({});
+    validateServiceDates(errors, fieldData, formData);
+    expect(dischargeDateSpy.called).to.be.false;
+    expect(entryDateSpy.called).to.be.false;
   });
 
-  context('when discharge date is before entry date', () => {
-    const spy = sinon.spy();
+  it('should set error message when discharge date is before entry date', () => {
```

Comment:
I think we should consider reorganizing these at some point to be in contexts for `should set error message when` and `should not set error message when` or something similar, but I think that extra scope is probably not necessary at the moment.